spec/factories/user.rb

```diff
@@ -15,6 +15,10 @@
     factory :admin do
       is_admin true
     end
+
+    factory :non_admin do
+      is_admin false
```

Comment:
It works, but in fact I think it's useless: you could simply set the `is_admin` flag attribute to `false` in the default `user` factory, and only override it in the `admin` one.
This would allow you to write:

```
create :user
# => create a non admin user

create :user, :admin
# => create an admin
```